host/host.go

```diff
@@ -124,12 +127,25 @@ func New(log zerolog.Logger, address string, port uint, options ...func(*Config)
 		return nil, fmt.Errorf("could not create libp2p host: %w", err)
 	}
 
+	if cfg.EnableP2PRelay {
+		log.Info().Msg("enabling p2p relay...")
+		_, err = relayv2.New(h)
+		if err != nil {
+			return nil, fmt.Errorf("could not create relay: %w", err)
+		}
+	}
+
 	host := Host{
 		log: log.With().Str("component", "host").Logger(),
 		cfg: cfg,
 	}
 	host.Host = h
 
+	log.Printf("PeerID: %s", host.ID().String())
+	for _, addr := range h.Addrs() {
+		log.Printf("Listening on: %s/p2p/%s\n", addr.String(), host.ID())
+	}
+
```

Comment:
These are probably debug helpers that were forgotten..? I think we should remove these..

We actually do log this exact thing in [main.go](https://github.com/blocklessnetwork/b7s/blob/main/cmd/node/main.go#L153).

Reply:
https://github.com/blocklessnetwork/b7s/pull/168/commits/eeda4400adeb8b9d33bee6d439b3a7cafa48afef